### DISCUSSION TITLE
ArgoCD source repo change

### DISCUSSION THREAD

**Discussion by luckpng** [argoproj/argo-cd#16378] [Q&A]
Hello everyone!

We're using ArgoCD + Azure Repos for GitOps but we want to change our source to Github. Is there anything that we need to worry about migrating this? I ask because I've heard that the cluster could be destroyed if we did this.

To be more clear:

today: ArgoCD pointing to the main branch on Azure Repos that where are our k8s manifest files (monorepo);
project: ArgoCD pointing to the main branch on Github (clone of Azure Repos) containing the same files/manifests of k8s.

**Comment by FernandoMiguel** [comment]
as long as your argo applications or applicationsets remain the same, the source doesnt really matter
